src/layers/normalise.jl

```diff
@@ -431,6 +431,10 @@ function InstanceNorm(chs::Int, λ=identity;
                     affine=false, track_stats=false,
                     ϵ=1f-5, momentum=0.1f0)
 
+  if track_stats
+    @warn("`track_stats=true` will be removed. Please use `track_stats=false` instead")
+  end
```

Comment:
Should this use `Base.depwarn`? Maybe changed to `force=true` on the last 0.13 release (otherwise it only prints during tests etc.)
```suggestion
  if track_stats
    Base.depwarn("`track_stats=true` will be removed from InstanceNorm in Flux 0.14. The default value is `track_stats=false`, which will work as before.", :InstanceNorm)
  end
```

Reply:
It does not be triggered `Base.depwarn` when I use. So I use a `@warn` for triggering a deprecation message. Could you give some tips?

Reply:
You may need to call `Base.depwarn("...", :InstanceNorm; force=true)` as suggested above.

Reply:
@ToucheSir Thanks for sharing your valuable tip. `force=flag` is really work properly.